packages/core/test/bot-helpers/uniswap-broker/UniswapV3Helpers.js

```diff
@@ -0,0 +1,156 @@
+// This file contains a number of useful helper functions. They were either written specifically for these unit tests,
```

Comment:
Could you point out which ones are pulled from uniswap and which ones we wrote? Just so we know what to review?

Reply:
all tests that were pulled from uniswapv3 were modified to work without ethers so everything was adapted at some level. Regarding reviewing them I think you can basically assume that they all "work" as expected as they came from uniswap v3 (plus modification for web3) except for:
1) `getCurrentPrice`
2) `getTickFromPrice`
3) `replaceLibraryBindingReferenceInArtitifact`

which I added specifically for these tests.